dev_combine_index_dag.py

```diff
@@ -116,8 +118,8 @@ def slackpostonsuccess(dag, **context):
         "BUCKET": AIRFLOW_DATA_BUCKET,
         "FOLDER": FCDAG.dag_id + "/" + TIMESTAMP,
         "SOLR_URL": SOLR_COLL_ENDPT,
-        "SOLR_AUTH_USER": SOLR_CONN.login,
-        "SOLR_AUTH_PASSWORD": SOLR_CONN.password,
+        "SOLR_AUTH_USER": SOLR_CONN.login or "",
+        "SOLR_AUTH_PASSWORD": SOLR_CONN.password or "",
```

Comment:
nice find. I've also seen use of `Variable.get("MY_VAR", default_var="default-value")`